packages/lavapack/src/builder-runtime.js

```diff
@@ -1,10 +1,18 @@
 const { join: pathJoin } = require('path')
 const { promises: { readFile, writeFile }} = require('fs')
 const { generateKernel, makeInitStatsHook } = require('lavamoat-core')
+const { getStrictScopeTerminatorShimSrc, replaceTemplateRequire } = require('lavamoat-core/src/generateKernel')
 
 module.exports = buildRuntime
 
+async function buildRuntimeCjs() {
+  const runtimeCjsTemplate = await readFile(pathJoin(__dirname, 'runtime-cjs-template.js'), 'utf8')
+  const sst = replaceTemplateRequire(runtimeCjsTemplate, 'strict-scope-terminator', getStrictScopeTerminatorShimSrc())
+  await writeFile(pathJoin(__dirname, 'runtime-cjs.js'), sst)
+}
+
 async function buildRuntime (opts = {}) {
+  await buildRuntimeCjs()
```

Comment:
Rather than nesting a `buildRuntimeCjs` inside (the otherwise similarly scoped) `buildRuntime` function, how about breaking out separating it?

For example, the existing `buildRuntime` could be renamed to `buildRuntimeES`, and then the `buildRuntime` could simply be:

```
   await Promise.all([buildRuntimeCJS(opts), buildRuntimeES(opts)]);
```

Reply:
3dd8329